modules/rostests/rosautotest/CWebServiceWinInet.cpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * PROJECT:     ReactOS Automatic Testing Utility
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Class implementing the WinInet version of the interface to the "testman" Web Service
+ * COPYRIGHT:   Copyright 2009-2015 Colin Finck (colin@reactos.org)
+ * COPYRIGHT:   Copyright 2025 Mark Jansen <mark.jansen@reactos.org>
+ */
+
+#include "precomp.h"
+
+/**
+ * Constructs a CWebService object and immediately establishes a connection to the "testman" Web Service.
+ */
+CWebServiceWinInet::CWebServiceWinInet()
+{
+    /* Zero-initialize variables */
+    m_hHTTP = NULL;
+    m_hHTTPRequest = NULL;
+
+    /* Establish an internet connection to the "testman" server */
+    m_hInet = InternetOpenW(L"rosautotest", INTERNET_OPEN_TYPE_PRECONFIG, NULL, NULL, 0);
+
+    if(!m_hInet)
+        FATAL("InternetOpenW failed\n");
+}
+
+/**
+ * Destructs a CWebService object and closes all connections to the Web Service.
+ */
+CWebServiceWinInet::~CWebServiceWinInet()
+{
+    if(m_hInet)
```

Comment:
```suggestion
    if (m_hInet)
```
etc. (the others below)